server/chat-plugins/scavengers.ts

```diff
@@ -1593,7 +1592,6 @@ const ScavengerCommands: ChatCommands = {
 		) {
 			return this.errorReply("You cannot edit the hints and answers if you are not the host.");
 		}
-
 		const [question, type, ...value] = target.split(',');
```

Comment:
```suggestion

				const [question, type, ...value] = target.split(',');
```

I think I'd prefer it back. I personally think whitespace after a `return` is more readable.